packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetailsMultiple.vue

```diff
@@ -1,21 +1,121 @@
 <template>
-  <div class="oc-mt-xl">
-    <div class="oc-flex space-info">
-      <oc-icon name="layout-grid" size="xxlarge" />
-      <p v-translate>Multiple spaces selected</p>
+  <div id="oc-spaces-details-multiple-sidebar">
+    <div class="spaces-preview oc-mb">
+      <div class="spaces-preview-body">
+        <oc-icon class="preview-icon" size="xxlarge" variation="passive" name="layout-grid" />
+        <p class="preview-text" data-testid="selectedFilesText" v-text="selectedSpacesString" />
+      </div>
+    </div>
+    <div>
+      <table class="details-table" :aria-label="detailsTableLabel">
+        <tr data-testid="filesCount">
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Total:')" />
+          <td v-text="totalSelectedSpaceQuotaTotal" />
+        </tr>
+        <tr data-testid="foldersCount">
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Remaining:')" />
+          <td v-text="totalSelectedSpaceQuotaRemaining" />
+        </tr>
+        <tr data-testid="spacesCount">
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Used:')" />
+          <td v-text="totalSelectedSpaceQuotaUsed" />
+        </tr>
+      </table>
     </div>
   </div>
 </template>
 <script lang="ts">
-import { defineComponent } from 'vue'
+import filesize from 'filesize'
+import { computed, defineComponent, PropType } from 'vue'
+import { SpaceResource } from 'web-client/src'
+import { useTranslations } from 'web-pkg/src'
 
 export default defineComponent({
-  name: 'SpaceDetailsMultiple'
+  name: 'SpaceDetailsMultiple',
+  props: {
+    selectedSpaces: {
+      type: Array as PropType<Array<SpaceResource>>,
+      required: true
+    }
+  },
+  setup(props) {
+    const { $ngettext, $gettextInterpolate } = useTranslations()
+    const totalSelectedSpaceQuotaTotal = computed(() => {
+      let total = 10
+      props.selectedSpaces.forEach((space) => {
+        total += space.spaceQuota.total
+      })
+      return filesize(total)
```

Comment:
TIL: We have a nice `formatFileSize` helper 🙂 Let's use that. It needs `$language.current` as second param, you can get that from `useTranslations()`.